mobile/src/main/java/org/openhab/habdroid/ui/OpenHABNotificationFragment.java

```diff
@@ -105,21 +89,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     public void onAttach(Activity activity) {
         super.onAttach(activity);
         Log.d(TAG, "onAttach()");
-        try {
-            mActivity = (OpenHABMainActivity) activity;
```

Comment:
When removing this assignment, the accesses to it need to be adapted to use getActivity().